setup.py

```diff
@@ -61,7 +61,7 @@ def append_local_version_label(public_version):
     'docker',
     'pyyaml',
     'pygithub',
-    'catkin_pkg >= 0.4.0',
+    'catkin_pkg >= 0.4.10',
```

Comment:
Commit summary: `get_build_type` -> `get_build_type()`

Commit body: `0.4.10 verion` -> `v0.4.10`

Reply:
Commit summary: get_build_type -> get_build_type() in a409be822b8e983dff5af518bb5a61c21a18ce0a (just a nit, obviously).